code/game/objects/items/storage/bags.dm

```diff
@@ -48,7 +48,7 @@
 	STR.max_w_class = WEIGHT_CLASS_SMALL
 	STR.max_combined_w_class = 30
 	STR.max_items = 30
-	STR.cant_hold = typecacheof(list(/obj/item/disk/nuclear))
+	STR.can_hold = typecacheof(list(/obj/item/ammo_casing, /obj/item/trash, /obj/item/dildo, /obj/item/electronics, /obj/item/circuitboard, /obj/item/reagent_containers/food, /obj/item/stock_parts, /obj/item/paper, /obj/item/reagent_containers/syringe, /obj/item/shard, /obj/item/pen))

```

Comment:
i'd say pills and bottles/soda cans should be added in too as well as water bottles things like that...
Hard press to not add in cardboard, and metal rods. As those are commanly found around as a trash item, broken lights are also a thing unless thats under trash.

Reply:
Soda cans are a subtype of /food so they're in.

Cardboard, rods and broken lights are good ideas though thanks.

Reply:
bottles are not, as well as things like coffie and such...